### DISCUSSION TITLE
how to get the result of a Transaction

### DISCUSSION THREAD

**Discussion by newcworld** [Elnaril/uniswap-universal-router-decoder#54] [Q&A]
I want to get Transaction's result
example:
address1 swap 1000 xxx token1 and get 222 yyy token2

**Comment by Elnaril** [comment]
Hi!
I'm closing this discussion as not not specific to this lib. Make sure you're familiar with general blockchain concepts and web3.py in particular before digging into specific protocols.
To answer your question, you need to request and analyse the transaction receipt.
